Makefile

```diff
@@ -185,19 +175,18 @@ docker-nydus-graphdriver-static:
 # refer to `misc/example/README.md` for details.
 all-static-release: docker-static all-contrib-static-release
 
-all-contrib-static-release: nydusify-static nydus-snapshotter-static ctr-remote-static \
+all-contrib-static-release: nydusify-static ctr-remote-static \
 			    nydus-overlayfs-static docker-nydus-graphdriver-static
 
-all-contrib-test: nydusify-test nydus-snapshotter-test ctr-remote-test \
-		  nydus-overlayfs-test docker-nydus-graphdriver-test
+all-contrib-test: nydusify-test ctr-remote-test \
+				nydus-overlayfs-test docker-nydus-graphdriver-test
 
 # https://www.gnu.org/software/make/manual/html_node/One-Shell.html
 .ONESHELL:
 docker-example: all-static-release
 	cp ${current_dir}/target-fusedev/${ARCH}-unknown-linux-musl/release/nydusd misc/example
 	cp ${current_dir}/target-fusedev/${ARCH}-unknown-linux-musl/release/nydus-image misc/example
 	cp contrib/nydusify/cmd/nydusify misc/example
-	cp contrib/nydus-snapshotter/bin/containerd-nydus-grpc misc/example
 	docker build -t nydus-rs-example misc/example
```

Comment:
docker-example is broken without `nydus-snapshotter`.

Reply:
True. How about releasing nydus-snapshotter in https://github.com/containerd/nydus-snapshotter and provide built binaries in its release page and download nydus-snapshotter when `make docker-example`

Reply:
Better if we can provide a docker image in https://github.com/containerd/nydus-snapshotter so that it is easier for users to deploy the remote snapshotter too.

Reply:
Could we mark this comment as solved since a docker image already is added into https://github.com/containerd/nydus-snapshotter

Reply:
We still need to figure out how to run a demo example here. But I agree it can be fixed later.

Reply:
Big agree